src/main/java/com/commercetools/sync/categories/utils/CategoryReferenceReplacementUtils.java

```diff
@@ -38,8 +38,8 @@
             .map(category -> {
```

Comment:
maybe we need to update java docs in here from references to resourceidentifiers like 
`the categories to replace their {@link ResourceIdentifier} ids with keys...`
 wdyt ?

Reply:
17ea123